Gruntfile.js

```diff
@@ -104,8 +104,8 @@ module.exports = function(grunt) {
 
         watch: {
             all: {
-                files: ['Gruntfile.js', 'src/**/*'],
-                tasks: ['jshint']
+                files: ['Gruntfile.js', 'src/**/*', 'test/**/*', 'assets/js/**/*'],
```

Comment:
Actually after merging your other PR, there's no need for the change to this file